x-pack/plugins/cloud_integrations/cloud_experiments/public/launch_darkly_client/launch_darkly_client.test.ts

```diff
@@ -114,18 +108,32 @@ describe('LaunchDarklyClient - browser', () => {
     });
 
     describe('getVariation', () => {
-      test('returns the default value if the user has not been defined', async () => {
```

Comment:
This won't work anymore because it waits resolve until user is defined.
My assumption is that launchDarklyClient won't be even created by the plugin if there won't be a user defined.